docs/api/RESTAPI.md

```diff
@@ -577,33 +617,7 @@ Returns a javascript file which will render a Gantt chart for the requested work
 * Workflows
 
 
-<a name="stats"></a>
-## Returns basic statistics from this Cromwell server.
-```
-GET /engine/{version}/stats
-```
-
-
-#### Parameters
-
-|Type|Name|Description|Schema|Default|
-|---|---|---|---|---|
-|**Path**|**version**  <br>*required*|Cromwell API Version|string|`"v1"`|
-
-
-#### Responses
-
-|HTTP Code|Description|Schema|
-|---|---|---|
-|**200**|Successful Request|[StatsResponse](#statsresponse)|
-
-
-#### Tags
-
-* Engine
-
-
-<a name="status"></a>
```

Comment:
good catch

Reply:
All of these weren't my changes. Over the next month or so look for an incoming PR to CI-verify that this file has been synced with cromwell.yaml.